slack_bolt/app/app.py

```diff
@@ -173,6 +175,8 @@ def message_hello(message, say):
             oauth_settings: The settings related to Slack app installation flow (OAuth flow)
             oauth_flow: Instantiated `slack_bolt.oauth.OAuthFlow`. This is always prioritized over oauth_settings.
             verification_token: Deprecated verification mechanism. This can used only for ssl_check requests.
+            listener_executor: Optional executor to run background tasks. If absent, a `ThreadPoolExecutor` will
```

Comment:
As the executor is not optional (some executor is required), I prefer "custom" for describing the given one 
```suggestion
            listener_executor: Custom executor to run background tasks. If absent, the default `ThreadPoolExecutor` will
```